api/v1beta2/foundationdbcluster_types.go

```diff
@@ -5,7 +5,7 @@ Licensed under the Apache License, Version 2.0 (the "License");
 you may not use this file except in compliance with the License.
 You may obtain a copy of the License at
 
-    http://www.apache.org/licenses/LICENSE-2.0
+	http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Is there a reason for this change?

Reply:
It's done by go fmt or something automatic.

Reply:
Could this be your emacs configuration, it seems like the spaces are replaced with one tab. If possible we should revert the change.